src/komponenter/common/arbeidsflate-lenker/arbeidsflate-lenker.ts

```diff
@@ -16,26 +17,26 @@ export const arbeidsflateLenker = (XP_BASE_URL: string): ArbeidsflateLenke[] =>
 export const personContextLenke = (XP_BASE_URL: string) => {
```

Comment:
Kunne vi ha satt return type ArbeidsflateLenke istedet? Så slipper vi å caste hver string som const?

```suggestion
export const personContextLenke = (XP_BASE_URL: string): ArbeidsflateLenke => {
```

Reply:
God ide! Jeg fiksa det nå 😄